src/common/hooks/useSelectedPOVersionMethods/useSelectedPOVersionMethods.js

```diff
@@ -0,0 +1,40 @@
+import { getFullName } from '@folio/stripes/util';
+import { useIntl } from 'react-intl';
+
+export const useSelectedPOVersionMethods = () => {
```

Comment:
please think about better name

Reply:
mb `useVersionHistoryFormatters`, kind of

Reply:
and all functions should be `formatSmth`

Reply:
@usavkov-epam does it make sense to move such hook to stripes-acq?

Reply:
Could you give a suggestion? Could not come up with anything better )

Reply:
Suppose it makes sense since similar places might result in the similar issue:

https://github.com/folio-org/ui-invoice/blob/master/src/common/hooks/useSelectedInvoiceVersion/useSelectedInvoiceVersion.js#L86-L89

https://github.com/folio-org/ui-invoice/blob/master/src/common/hooks/useSelectedInvoiceLineVersion/useSelectedInvoiceLineVersion.js#L55-L57

https://github.com/folio-org/ui-organizations/blob/master/src/Organizations/OrganizationVersion/hooks/useSelectedOrganizationVersion/useSelectedOrganizationVersion.js#L102-L106